### PR TITLE
Jb55177

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Under remotelinux they also omit '--' https://github.com/sailfishos/sailfish-qtcreator/blob/master/src/plugins/remotelinux/remotelinuxsignaloperation.cpp#L52

File: src/tools/sfdk/remoteprocess.cpp [hunk 1]
```diff
@@ -240,7 +240,7 @@ void RemoteProcess::kill(const QString &signal, void (RemoteProcess::*callback)(
         m_killRunner.reset(nullptr);
     });
 
-    QString killCommand = QString("kill -%1 -- -%2 %2").arg(signal).arg(m_processId);
+    QString killCommand = QString("kill -%1 -%2 %2").arg(signal).arg(m_processId);
```
Comment (vigejolla) [inline src/tools/sfdk/remoteprocess.cpp:243]: Why is the process ID there twice, with and without a preceding minus?

Comment (martyone) [reply src/tools/sfdk/remoteprocess.cpp:243]: Negation of a PID results in killing the process group the process leads.

Comment (martyone) [reply src/tools/sfdk/remoteprocess.cpp:243]: Negation of a PID results in killing the process group the process leads.

Comment (vigejolla) [reply src/tools/sfdk/remoteprocess.cpp:243]: TIL :)

**Pull request merged**